opencti-platform/opencti-front/src/private/components/data/csvMapper/CsvMapperForm.tsx

```diff
@@ -213,11 +224,20 @@ const CsvMapperForm: FunctionComponent<CsvMapperFormProps> = ({
                   <Typography>{t('Semicolon')}</Typography>
                 </div>
               </div>
-              <div className={classes.center} style={{ marginTop: 20 }}>
+              <div className={`${classes.center} ${classes.marginTop}`}>
+                  <Field
+                    component={TextField}
+                    type="skipLineChar"
```

Comment:
I don't understand this line : type="skipLineChar"

Reply:
Me neither